File: server/tests/fixtures/alpha_rule_triggers/java/errors_broad_catch.java
// Should trigger: errors.broad_catch
public class BroadCatch {
    public void process() {
        try {
            riskyOperation();
        } catch (Exception e) {  // Too broad!
            log(e);
        }
    }
    
    private void riskyOperation() throws Exception {}
    private void log(Exception e) {}
}

File: server/tests/fixtures/alpha_rule_triggers/csharp/errors_broad_catch.cs
// Should trigger: errors.broad_catch
public class BroadCatch {
    public void Process() {
        try {
            RiskyOperation();
        } catch (Exception e) {  // Too broad!
            Log(e);
        }
    }
    
    private void RiskyOperation() {}
    private void Log(Exception e) {}
}
